doc/source/whatsnew/v2.2.0.rst

```diff
@@ -390,6 +390,7 @@ Styler
 
 Other
 ^^^^^
+- Bug in :class:`Series` constructor raising DeprecationWarning when ``index`` is a list of :class:`Series` (:issue:`55228`)
```

Comment:
Can you move this to the v2.1.3.rst file?

Reply:
Sure.